cmd/migrate.go

```diff
@@ -41,6 +43,12 @@ func NewMigrateCommand() *cobra.Command {
 		viper.AddConfigPath(path)
 	}
 
+	err := viper.ReadInConfig()
+	if err == nil {
+		viper.SetDefault(datastoreEngineFlag, viper.Get(datastoreEngineConf))
+		viper.SetDefault(datastoreURIFlag, viper.Get(datastoreURIConf))
+	}
```

Comment:
```suggestion
	err := viper.ReadInConfig()
	if err == nil {
		if !viper.IsSet(datastoreEngineConf) {
			log.Fatalf("missing required field: %v", datastoreEngineFlag)
		}
		if !viper.IsSet(datastoreURIConf) {
			log.Fatalf("missing required field: %v", datastoreURIFlag)
		}
		viper.SetDefault(datastoreEngineFlag, viper.Get(datastoreEngineConf))
		viper.SetDefault(datastoreURIFlag, viper.Get(datastoreURIConf))
	}
```

Reply:
Well it is not checked either by the `run` command. Setting the `engine` to `postgres` there without an `uri` produces the following output:

    $ cat ~/.openfga/config.yaml
    datastore:
      engine: postgres
    
    $ ./openfga run             
    2023-03-31T18:13:59.674+0200	INFO	🧪 experimental features enabled: []
    2023-03-31T18:13:59.675+0200	INFO	waiting for postgres	{"attempt": 1}
    2023-03-31T18:14:00.229+0200	INFO	waiting for postgres	{"attempt": 2}
    2023-03-31T18:14:01.311+0200	INFO	waiting for postgres	{"attempt": 3}
    2023-03-31T18:14:02.623+0200	INFO	waiting for postgres	{"attempt": 4}
    2023-03-31T18:14:04.207+0200	INFO	waiting for postgres	{"attempt": 5}
    2023-03-31T18:14:06.550+0200	INFO	waiting for postgres	{"attempt": 6}
    2023-03-31T18:14:11.062+0200	INFO	waiting for postgres	{"attempt": 7}